src/pypgstac/tests/hydration/test_dehydrate_pg.py

```diff
@@ -14,7 +16,6 @@ class TestDehydratePG(TDehydrate):
     @contextmanager
     def db(self) -> Generator:
         """Set up database connection."""
-        print("Setting up db.")
```

Comment:
👋

Reply:
Believe it or not, I don't think that one was me!

Reply:
oh yeah I do print in testing too :D